### PR TITLE
[MIG] shopinvader_customer_price_wishlist: Migration to 14.0

### PR DESCRIPTION
**Pull request by chafique-delli**
Depends on:

- [x] https://github.com/shopinvader/odoo-shopinvader/pull/895

### CODE REVIEW

File: oca_dependencies.txt [hunk 1]
```diff
@@ -1 +1 @@
-# See https://github.com/OCA/odoo-community.org/blob/master/website/Contribution/CONTRIBUTING.rst#oca_dependencies-txt
```
Comment (simahawk) [inline oca_dependencies.txt]: IDT think we need this as we use MQT=PIP here. You could trash the file.

**Review by simahawk — CHANGES_REQUESTED** [review]

**Comment by simahawk** [comment]
@kevinkhao can you rebase? Should fix tests

**Comment by kevinkhao** [comment]
renamed testcase here https://github.com/shopinvader/odoo-shopinvader/pull/1142

**Comment by simahawk** [comment]
@kevinkhao #1142 is getting merged

**Comment by simahawk** [comment]
@kevinkhao could you check the build?
```
2021-10-08 09:52:20,977 422 ERROR odoo odoo.addons.shopinvader_customer_price_wishlist.tests.test_wishlist_price: ERROR: WishlistCase.test_jsonify_custom_pricelist
Traceback (most recent call last):
  File "/__w/odoo-shopinvader/odoo-shopinvader/shopinvader_customer_price_wishlist/tests/test_wishlist_price.py", line 53, in test_jsonify_custom_pricelist
    self.discount_pricelist, self.backend._get_fiscal_position(self.partner)
  File "/__w/odoo-shopinvader/odoo-shopinvader/shopinvader_product_price_tax/models/shopinvader_variant.py", line 13, in _get_price
    qty=qty, pricelist=pricelist, fposition=fposition, company=company
  File "/__w/odoo-shopinvader/odoo-shopinvader/shopinvader/models/shopinvader_variant.py", line 242, in _get_price
    if pricelist and pricelist.discount_policy == "without_discount":
AttributeError: 'account.fiscal.position' object has no attribute 'discount_policy'
```

**Comment by kevinkhao** [comment]
function signature changed with named arguments
I put the named arguments

**Review by simahawk — APPROVED** [review]

**Pull request merged**